actors/evm/src/interpreter/instructions/control.rs

```diff
@@ -309,4 +309,38 @@ mod tests {
             assert_eq!(m.pc, 1, "pc has not advanced to 1");
         }
     }
+
+    #[test]
```

Comment:
ditto: can we test out-of-bounds behavior?

Reply:
Same here.

Reply:
- RETURNDATACOPY fails if the `offset + length` exceeds the end of the return data buffer.

Also make sure to check for integer overflows/underflows.

Reply:
added some edge case tests.